java/api/answer/JSON.java

```diff
@@ -59,6 +59,10 @@ private static JSON of(JsonValue value) {
         }
     }
 
+    private static java.lang.String indent(java.lang.String string) {
```

Comment:
It was autoformatter lol

I haven't changed the class as it suits my current needs. I could remove some potentially redundant stuff, but we'll be working on different fetch options later, so we'll return here anyway, and there is no point in cleaning it right now. To my mind.